applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr_field/ProjectParameters.json

```
@@ -0,0 +1,263 @@
+    "number_cycles": 100,
+    "reduction_factor": 0.5,
+    "increase_factor": 2.0,
+    "desired_iterations": 4,
+    "max_radius_factor": 10.0,
+    "min_radius_factor": 0.1,
+    "calculate_reactions": false,
+    "max_line_search_iterations": 5,
+    "first_alpha_value": 0.5,
+    "second_alpha_value": 1.0,
+    "min_alpha": 0.1,
+    "max_alpha": 2.0,
+    "line_search_tolerance": 0.5,
+    "rotation_dofs": true,
+    "linear_solver_settings": {
+      "solver_type": "LinearSolversApplication.sparse_lu",
+      "scaling": true
+    },
+    "problem_domain_sub_model_part_list": [
+      "Soil-3"
+    ],
+    "processes_sub_model_part_list": [
+      "Displacement_Geometry",
+      "Gravity_Loading-Initial_saturated",
+      "Base_Displacement_Fixed",
+      "Fluid_Pressure-Initial_saturated",
+      "Lower_Drainage_Fluid_Pressure"
+    ],
+    "body_domain_sub_model_part_list": [
+      "Soil-3"
+    ],
+    "newmark_beta": 0.25,
+    "newmark_gamma": 0.5,
+    "newmark_theta": 0.5,
+    "rayleigh_m": 0.0,
+    "rayleigh_k": 0.0
+  },
+  "output_processes": {
+    "gid_output": [
+      {
+        "python_module": "gid_output_process",
+        "kratos_module": "KratosMultiphysics",
+        "process_name": "GiDOutputProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.porous_computational_model_part",
+          "output_name": "mesh0",
+          "postprocess_parameters": {
+            "result_file_configuration": {
+              "gidpost_flags": {
+                "WriteDeformedMeshFlag": "WriteUndeformed",
+                "WriteConditionsFlag": "WriteElementsOnly",
+                "GiDPostMode": "GiD_PostAscii",
+                "MultiFileFlag": "SingleFile"
+              },
+              "file_label": "step",
+              "output_control_type": "step",
+              "output_interval": 1,
+              "body_output": true,
+              "node_output": false,
+              "skin_output": false,
+              "plane_output": [],
+              "nodal_results": [
+                "DISPLACEMENT",
+                "TOTAL_DISPLACEMENT",
+                "WATER_PRESSURE"
+              ],
+              "gauss_point_results": [
+                "CAUCHY_STRESS_TENSOR",
+                "TOTAL_STRESS_TENSOR"
+              ]
+            },
+            "point_data_configuration": []
+          }
+        }
+      }
+    ]
+  },
+  "processes": {
+    "constraints_process_list": [
+      {
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name": "ApplyVectorConstraintTableProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.Displacement_Geometry",
+          "variable_name": "DISPLACEMENT",
+          "active": [
+            true,
+            false,
+            true
+          ],
+          "is_fixed": [
+            true,
+            false,
+            true
+          ],
```

```suggestion
          "is_fixed": [true, false, true],
```
I think the file becomes more clear when putting these 'arrays' on a single line

---

I agree. This repeats in a lot of our test files. Is there an automated way to do this on all files? A .json prettifier or something similar that will work?

---

Done